app/pages/SearchResultsPage/SearchResultsPage.tsx

```diff
@@ -82,17 +102,16 @@ const InnerSearchReslts = ({
       }}
       createURL={(state: any) => `search?${qs.stringify(state)}`}
     >
-      {userLocation ? (
```

Comment:
If a user refuses to share their location, we set userLocation coords to a default SF location. Thus, we will always have a userLocation and this check is moot